dpctl/tests/test_sycl_platform.py

```diff
@@ -87,6 +87,11 @@ def check_repr(platform):
     assert r != ""
 
 
+def check_default_context(platform):
+    r = platform.default_context
+    assert type(r) is dpctl.SyclContext
```

Comment:
Do we ever expect `r` to be None?

Reply:
Guess not, but can run into an Exception. The test case can handle that?

Reply:
Yes, an exception would get raised and pytest will report it